From cf8f4d4fc33793fd8b5e263bf0fa331159c9c529 Mon Sep 17 00:00:00 2001
From: Will Thompson <wjt@endlessos.org>
Date: Fri, 29 Apr 2022 12:46:00 +0100
Subject: [PATCH 09/13] metainfo: Add <translation> tag

This allows the software repository, such as Debian, Fedora or Flathub,
to extract data about the app's translation coverage; in turn, the
software centre (such as GNOME Software) can indicate to the user
whether the software is available in their language.

See
https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-translation
for more details.
---
 gobby-0.5.metainfo.xml.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/gobby-0.5.metainfo.xml.in b/gobby-0.5.metainfo.xml.in
index e76ecb9..03ae16b 100644
--- a/gobby-0.5.metainfo.xml.in
+++ b/gobby-0.5.metainfo.xml.in
@@ -29,6 +29,8 @@
   <url type="homepage">https://gobby.github.io/</url>
   <url type="bugtracker">https://github.com/gobby/gobby/issues</url>
 
+  <translation type="gettext">gobby05</translation>
+
   <releases>
     <release version="0.6" date="2021-01-31">
       <description>
-- 
2.30.2

